regen_doc.g

```diff
@@ -74,6 +135,17 @@ local xmlfile, meth;
     for meth in Set(RecNames(db)) do
         AppendTo(xmlfile, "<Subsection Label=\"", meth, "\">\n");
         AppendTo(xmlfile, "<Heading><C>", meth, "</C></Heading>\n");
+        # Where is this method used?
+        AppendTo(xmlfile, "This method is ");
+        dbsWhichUseMethod := DbsWhichUseMethod(db, meth);
+        if IsEmpty(dbsWhichUseMethod) then
+            AppendTo(xmlfile, "unused!");
+        else
+            AppendTo(xmlfile, "used for recognizing ",
+                     JoinStringsWithSeparator(dbsWhichUseMethod, ", "),
```

Comment:
```suggestion
                     JoinStringsWithSeparator(dbsWhichUseMethod, " and "),
```
Even better would be to replace just the final "," by "and", but the above IMHO is already an improvement.